app/Http/Kernel.php

```diff
@@ -37,7 +37,7 @@ class Kernel extends HttpKernel
             \Illuminate\Session\Middleware\StartSession::class,
             // \Illuminate\Session\Middleware\AuthenticateSession::class,
             \Illuminate\View\Middleware\ShareErrorsFromSession::class,
-            \App\Http\Middleware\VerifyCsrfToken::class,
+           // \App\Http\Middleware\VerifyCsrfToken::class,
```

Comment:
Why u remove csrf token ?

Reply:
i must've forgotten to uncomment it
i commented it because i was trying code on postman and it kept causing exceptions